app/views/posts/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1> Here is a single Post Page of a User <h1>
+<div>
+    <div class="d-flex justify-content-between shadow p-3 mb-5 bg-body rounded">
+        Post <%= @post.id %> by <%= @post.author.name %>
+        <div>
+            <span>Comments: <%= @post.comments_count %> </span> <span> Likes: <%= @post.likes_count %> </span>
+        </div>
+    </div>
+
+    <p class="shadow p-3 mb-5 bg-body rounded"> <%= @post.text%> </p>    
+</div>
+
+<div class="d-flex flex-column p-3 mb-5 bg-body rounded">
+    <% @post.comments.each do |comment|%>
+        <span> <%= comment.author.name %>: <%= comment.text%> </span>
+    <% end %>
+</div>
```

Comment:
- Optional
Kindly fix the border issues with this segment of your application. The border is not visible currently.
⬇️
![image](https://user-images.githubusercontent.com/33268927/205886883-03cb0bc5-0cd5-4ffc-b73f-fa2e7db6f707.png)

Reply:
Alright I would work on it. Thank you @muneebulrehman